test/test_reductions.py

```diff
@@ -2860,10 +2860,106 @@ def test_histogram(self, device, dtype):
             self.assertEqual(actual_hist, expected_hist)
             self.assertEqual(actual_bin_edges, expected_bin_edges)
 
+    """
+    Runs torch.histogramdd and numpy.histogramdd on the specified input parameters
+    and asserts that their output is equal.
+    """
+    def _test_histogramdd_numpy(self, t, bins, bin_range, weights, density):
+        def to_np(t):
```

Comment:
Note for the future: we should think about making a canonical `to_np` function for torch.testing._internal